run-qa-checks

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# TODO: remove --skip-iflake8
```

Comment:
`Black` has a line length of 88 (which is being used in `openwisp-qa-format`) and is not compliant with `openwisp-qa-check` which checks for string length <79.

Opened https://github.com/openwisp/openwisp-utils/issues/203 for this